### PR TITLE
Fix: Continue importing from file if single emoji import fails

### LINKED ISSUE #14460: fix: missing emojis in meta.json interrupts emoji import
### 💡 Summary

if a `meta.json` file in an emoji zip references a file not present in the zip, the import process will log an error and exit, interrupting the import

### 🥰 Expected Behavior

the remaining emojis should be imported without error

### 🤬 Actual Behavior

only the emojis up to the missing one are imported

### 📝 Steps to Reproduce

_No response_

### 💻 Frontend Environment

_No response_

### 🛰 Backend Environment (for server admin)

```markdown
* Misskey: all versions back to at least commit b75184e
```

### Do you want to address this bug yourself?

- [X] Yes, I will patch the bug myself and send a pull request

### PR DESCRIPTION
**Pull request by winrg**
## What

Add a try/catch block to the DriveService.addFile call to allow other emojis to be processed in the case of a missing one.

## Why

There are potentially still good emojis in a file after one bad one, so interrupting the import process is bad behavior.

## Checklist
- [x] Read the [contribution guide](https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md)
- [x] Test working in a local environment
- [x] (If needed) Update CHANGELOG.md

Fixes #14460

### CODE REVIEW

File: packages/backend/src/queue/processors/ImportCustomEmojisProcessorService.ts [hunk 1]
```diff
@@ -87,23 +87,30 @@ export class ImportCustomEmojisProcessorService {
...
+  					name: emojiInfo.name,
+  					category: emojiInfo.category,
+  					host: null,
+  					aliases: emojiInfo.aliases,
+  					driveFile,
+  					license: emojiInfo.license,
+  					isSensitive: emojiInfo.isSensitive,
+  					localOnly: emojiInfo.localOnly,
+  					roleIdsThatCanBeUsedThisEmojiAsReaction: [],
+  				});
```
Comment (kakkokari-gtyih) [inline packages/backend/src/queue/processors/ImportCustomEmojisProcessorService.ts]:
```suggestion
  					await this.customEmojiService.add({
  						name: emojiInfo.name,
	  					category: emojiInfo.category,
	  					host: null,
  						aliases: emojiInfo.aliases,
  						driveFile,
	  					license: emojiInfo.license,
  						isSensitive: emojiInfo.isSensitive,
  						localOnly: emojiInfo.localOnly,
  						roleIdsThatCanBeUsedThisEmojiAsReaction: [],
  					});
```

The indentation needs to be raised one level

Comment (KisaragiEffective) [reply packages/backend/src/queue/processors/ImportCustomEmojisProcessorService.ts]: also Misskey uses only TABs

File: packages/backend/src/queue/processors/ImportCustomEmojisProcessorService.ts [hunk 2]
```diff
@@ -87,23 +87,30 @@ export class ImportCustomEmojisProcessorService {
...
+  						name: emojiInfo.name,
+	  					category: emojiInfo.category,
+	  					host: null,
+  						aliases: emojiInfo.aliases,
+  						driveFile,
+	  					license: emojiInfo.license,
+  						isSensitive: emojiInfo.isSensitive,
+  						localOnly: emojiInfo.localOnly,
+  						roleIdsThatCanBeUsedThisEmojiAsReaction: [],
+  					});
```
Comment (KisaragiEffective) [inline packages/backend/src/queue/processors/ImportCustomEmojisProcessorService.ts]: Please remove two spaces

**Comment by KisaragiEffective** [comment]
Does this close #14460 ?
Please [link](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) it in the description from here if it does.

**Comment by winrg** [comment]
I don't have the option to do so in the sidebar

**Comment by winrg** [comment]
i have done so

**Review by KisaragiEffective — APPROVED** [review]
LGTM, thank you

**Comment by syuilo** [comment]
🙏🏻

**Pull request merged**